GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -862,7 +860,7 @@ private void ShowLoading(bool showSpinner = true)
         ///  Queries git for the new set of revisions and refreshes the grid.
         /// </summary>
         /// <exception cref="AggregateException"></exception>
-        public void PerformRefreshRevisions(Func<RefsFilter, IReadOnlyList<IGitRef>> getRefs = null)
+        public void PerformRefreshRevisions(Func<RefsFilter, IReadOnlyList<IGitRef>> getRefs = null, bool updateSidepanel = false)
```

Comment:
This name implies this control needs to know about some other unrelated control, which is undesirable.
```suggestion
        public void PerformRefreshRevisions(Func<RefsFilter, IReadOnlyList<IGitRef>> getRefs = null, bool forceRefresh = false)
```

Also please add the xmldoc explaining the parameter.

Reply:
done